src/hello-world/variables-and-values.md

```diff
@@ -1,4 +1,25 @@
-# Scalar Types
+---
+minutes: 10
+existing course material:
+- basic-syntax/scalar-types.md
+- basic-syntax/variables.md
+---
+
+# Variables and Values
+
+Rust provides type safety via static typing. Variable bindings are made with
+`let`:
+
+```rust,editable
+fn main() {
+    let x: i32 = 10;
+    println!("x: {x}");
+    // x = 20;
+    // println!("x: {x}");
+}
+```
+
+## Scalar Values
```

Comment:
This heading seems wrong for a table of types and literals. Maybe we should make this slide "Variables, Values, and Types"?